ansible_base/rbac/models.py

```diff
@@ -198,7 +198,7 @@ def give_or_remove_global_permission(self, actor, giving=True):
             kwargs = dict(object_role=None, team=actor, role_definition=self)
             cls = RoleTeamAssignment
         else:
-            raise RuntimeError(f'Cannot give permission to {actor}, must be a user or team')
+            raise RuntimeError(f'Cannot give or remove permission to {actor}, must be a user or team')
```

Comment:
```suggestion
            action = "give" if giving else "remove"
            raise RuntimeError(f'Cannot {action} permission to {actor}, must be a user or team')
```

Reply:
I would do 
```py
f'Cannot {giving and "give" or "remove"} permission to {actor}, must be a user or team'
```

But others may think it is cryptic

Reply:
fine by me

```
>>> giving = False
>>> actor = 'bob'
>>> f'Cannot {giving and "give" or "remove"} permission to {actor}, must be a user or team'
'Cannot remove permission to bob, must be a user or team'
>>> 
>>> giving = True
>>> f'Cannot {giving and "give" or "remove"} permission to {actor}, must be a user or team'
'Cannot give permission to bob, must be a user or team'
```